README.md

```diff
@@ -93,6 +93,13 @@ or download a job asset:
 
 ```sh
 sl downloadJobAsset 690c5877710c422d8be4c622b40c747f video.mp4 --filepath ./video.mp4
+```
+
+or start Sauce Connect Proxy in EU datacenter:
+
+```sh
+sl sc --region eu --sePort 4445
```

Comment:
I see that you provide the options in the constants, is there a way to link to that in the readme, making the options clear and prevent double typing?

Reply:
You can see all available options via:

```sh
$ sl sc --help
```

I will add a note to the readme. I don't think documenting this in the readme is a good solution when we could just link to the wiki page.